src/com/lftechnology/java/training/sanish/application/component/EmployeeHelper.java

```diff
@@ -0,0 +1,416 @@
+                    LOGGER.log(Level.INFO, message);
+                    edit = UserInput.getString(inputScanner);
+                    if (!edit.equals(Constants.YES_OPTION) && !edit.equals(Constants.FULL_YES_OPTION)) {
+                        break;
+                    }
+                }
+            }
+        }
+    }
+
+    /**
+     * Change email of user
+     *
+     * @param inputScanner {@link Scanner}
+     * @param user         {@link User}
+     * @author Sanish Maharjan <sanishmaharjan@lftechnology.com>
+     */
+    public static void changeUserEmail(Scanner inputScanner, User user) {
+        String message = "\n" + Constants.USER_EMAIL_LABEL + user.getEmail() + "\n>>" + Constants.CHANGE_EMAIL_CONFORM_MSG;
+        LOGGER.log(Level.INFO, message);
+        String edit = UserInput.getString(inputScanner);
+        if (edit.equals(Constants.YES_OPTION) || edit.equals(Constants.FULL_YES_OPTION)) {
+            int updateRow = 0;
+            String email;
+            String SetStatement;
+            String whereCondition = "userId=?";
+            UserDao userDao = new UserDao();
+            while (updateRow == 0) {
+                message = "\n >>" + Constants.TYPE_EMAIL_LABEL;
+                LOGGER.log(Level.INFO, message);
+                email = UserInput.getString(inputScanner);
+                DateFormat dateFormat = new SimpleDateFormat(Constants.DB_DATE_FORMAT);
+                Date date = new Date();
+                SetStatement = "email=?, modifiedAt=?";
+                updateRow = userDao.update(whereCondition, SetStatement, email, dateFormat.format(date), user.getUserId());
+                if (updateRow == 0) {
+                    message = "\n" + Constants.ERROR_MSG_LABEL + Constants.FAIL_UPDATE_EMAIL_MSG + Constants.TRY_AGAIN_MSG;
+                    LOGGER.log(Level.INFO, message);
+                    edit = UserInput.getString(inputScanner);
+                    if (!edit.equals(Constants.YES_OPTION) && !edit.equals(Constants.FULL_YES_OPTION)) {
+                        break;
+                    }
+                }
+            }
+        }
+    }
+
+    /**
+     * Change full name of user
+     *
+     * @param inputScanner {@link Scanner}
+     * @param employee     {@link Employee}
+     * @author Sanish Maharjan <sanishmaharjan@lftechnology.com>
+     */
+    public static void changeFullName(Scanner inputScanner, Employee employee) {
+        String message = "\n" + Constants.USER_FULL_NAME_LABEL + employee.getFullName() + "\n>>" + Constants.CHANGE_FULL_NAME_CONFORM_MSG;
+        LOGGER.log(Level.INFO, message);
+        String edit = UserInput.getString(inputScanner);
+        if (edit.equals(Constants.YES_OPTION) || edit.equals(Constants.FULL_YES_OPTION)) {
+            int updateRow = 0;
+            String fullName;
+            String SetStatement;
+            String whereCondition = "employeeId=?";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "employeeId=?" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)